enhancements/20230316-sriov-network-devices.md

```diff
@@ -0,0 +1,138 @@
+# SR-IOV Network Devices
+
+## Summary
+
+As of harvester v1.1.x, support for PCI device passthrough to workload VMs has been introduced. 
+
+Building on top of this functionality, we can now extend support for passing through network virtual functions (VFs) to workload VMs.
+
+### Related Issues
+
+https://github.com/harvester/harvester/issues/2763
+
+## Motivation
+
+### Goals
+
+- Enable Harvester to identify SR-IOV capable network devices.
+
+- Allow users to define the number of virtual functions (VFs) on a network device.
+
+- Support passthrough of the newly created network virtual functions as any other PCI devices.
+
+### Non-goals
+
+Custom driver installation for network devices.
+
+## Proposal
+
+### User Stories
+
+#### Configure and passthrough virtual network functions
+
+### User Experience In Detail
+A harvester user wants to leverage SR-IOV capable virtual network functions for workload VM's running in a cluster.
+
+Currently this is not possible as the user can only passthrough the entire network physical function to a VM, and this cannot be shared among multiple workload VM's running on the same node.
```

Comment:
```suggestion
Leveraging SR-IOV capable virtual network functions for workload VMs running in a cluster is currently not possible since the user can only pass through the entire network physical function to a VM, and this can't be shared among multiple workload VMs running on the same node.
```